ci/start_init_test.sh

```diff
@@ -186,16 +189,15 @@ verify_controller_rep_state() {
 			rep_index=`expr $rep_index + 1`
 		done
 		i=`expr $i + 1`
-		sleep 4
+		sleep 2
 	done
 	echo $3 " -- failed"
 	collect_logs_and_exit
 }
 
 # start_controller CONTROLLER_IP
 start_controller() {
-	controller_id=$(docker run -d --net stg-net --ip "$1" -P --expose 3260 --expose 9501 --expose 9502-9504 $JI \
-		launch controller --frontend gotgt --frontendIP "$1" "$2")
+	controller_id=$(docker run -d --net stg-net --ip $1 -P --expose 3260 --expose 9501 --expose 9502-9504 $JI env REPLICATION_FACTOR="$3" launch controller --frontend gotgt --frontendIP "$1" "$2")
```

Comment:
is there any problem in having the '\', i.e., splitting this big single line to multiple small ones ?